Meadowlark-js/backends/meadowlark-mongodb-backend/test/integration/locking/Delete.test.ts

```diff
@@ -164,18 +194,40 @@ describe('given a delete concurrent with an insert referencing the to-be-deleted
     // **** The insert of AcademicWeek document should have been successful
     expect(upsertedCount).toBe(1);
 
+    // Adds the academic week and the, to be updated, school to the concurrency collection.
+    const concurrencyDocumentsAcademicWeek: ConcurrencyDocument[] = [];
+    concurrencyDocumentsAcademicWeek.push({ _id: documentUuid });
+
+    const schoolDocumentUuid: WithId<MeadowlarkDocument> | null = await mongoDocumentCollection.findOne(
+      { _id: schoolMeadowlarkId },
+      { projection: { documentUuid: 1 } },
+    );
+    if (schoolDocumentUuid) concurrencyDocumentsAcademicWeek.push({ _id: schoolDocumentUuid?.documentUuid });
+
+    await lockDocuments(mongoConcurrencyCollection, concurrencyDocumentsAcademicWeek, upsertSession);
+
     // ----
     // End transaction to insert the AcademicWeek document
     // ----
     await upsertSession.commitTransaction();
 
+    const concurrencyDocumentsSchool: ConcurrencyDocument[] = [];
+    concurrencyDocumentsSchool.push({
+      _id: schoolDocument.documentUuid,
+    });
+
     // Try deleting the School document - should fail thanks to AcademicWeek's read-for-write lock
     try {
-      await mongoCollection.deleteOne({ _id: schoolMeadowlarkId }, { session: deleteSession });
+      await lockDocuments(mongoConcurrencyCollection, concurrencyDocumentsSchool, deleteSession);
+
+      await mongoDocumentCollection.deleteOne({ _id: schoolMeadowlarkId }, { session: deleteSession });
     } catch (e) {
-      expect(e).toMatchInlineSnapshot(
-        `[MongoServerError: WriteConflict error: this operation conflicted with another operation. Please retry your operation or multi-document transaction.]`,
+      expect(e.message).toContain(
```

Comment:
Here it's better to use expect(e).toMatchInlineSnapshot('') that way it can be automatically updated